_inc/lib/admin-pages/class.jetpack-react-page.php

```diff
@@ -387,6 +387,7 @@ function jetpack_current_user_data() {
 			'edit_posts'         => current_user_can( 'edit_posts' ),
 			'publish_posts'      => current_user_can( 'publish_posts' ),
 			'manage_options'     => current_user_can( 'manage_options' ),
+			'customize'          => current_user_can( 'customize' ),
```

Comment:
What's the difference between `manage_options` and `customize` in terms of permissions? Could we use `manage_options` here?

Reply:
That option is specifically to ensures the user has access to the Customizer.